### PR TITLE
DOB Fix + Tweaks

### PR DESCRIPTION
**Pull request by anthonyvelazquez**
DOB Fix + Tweaks

### CODE REVIEW

**Comment by yoomlam** [comment]
[Slack thread](https://dsva.slack.com/archives/C01Q7979Z7D/p1668622315342869?thread_ts=1668617826.587689&cid=C01Q7979Z7D)

**Review by tjheffner — DISMISSED** [review]

**Comment by yoomlam** [comment]
@anthonyvelazquez How can we verify "There should be 18pt space before H1"?

File: service-python/pdfgenerator/src/lib/templates/hypertension/about.html [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<h3>About this Document</h3>
+<h1 style="font-size: 21px; margin-top: 24px;">About this Document</h1>
```
Comment (anthonyvelazquez) [inline service-python/pdfgenerator/src/lib/templates/hypertension/about.html:1]: @yoomlam missed pushing a commit. Now the spacing is in place. Not sure theres a way to verify outside of comparing both PDFs visually or checking the code so it has a `margin-top: 24px`

File: service-python/pdfgenerator/src/lib/templates/shared/intro.html [hunk 2]
```diff
@@ -81,9 +81,9 @@ <h1>VHA {{document_type|capitalize}} Data Summary</h1>
   <i>Generated automatically on {{timestamp.strftime('%m/%d/%Y')}} at {{timestamp.strftime('%H:%M %p %Z')}}</i>
   <br /><br />
   
-  <p class="m-0">{{veteran_info.first}} {{veteran_info.middle}} {{veteran_info.last}} {{veteran_info.suffix}}</p>
+  <h1 class="m-0" style="font-size: 21px; margin-top: 24px;">{{veteran_info.first}} {{veteran_info.middle}} {{veteran_info.last}} {{veteran_info.suffix}}</h1>
 
-  <p>DOB: {{veteran_info.birthdate}}</p>
+  <p>DOB: {{veteran_info.birthdate.strftime('%m/%d/%Y')}}</p>
```
Comment (yoomlam) [inline service-python/pdfgenerator/src/lib/templates/shared/intro.html:86]: Waiting on response from Shannon [here](https://dsva.slack.com/archives/C01Q7979Z7D/p1668628435307339?thread_ts=1668617826.587689&cid=C01Q7979Z7D)

**Review by yoomlam — APPROVED** [review]
Looks good! Pending Shannon's reply

**Review by tjheffner — APPROVED** [review]

File: service-python/pdfgenerator/src/lib/templates/shared/intro.html (hunk 2 above)
Comment (yoomlam) [reply service-python/pdfgenerator/src/lib/templates/shared/intro.html:86]: DOB format is good (for now)! No change needed. Please merge, including to the `main` branch being used for end-to-end testing.

**Pull request merged**